drivers/wd/onitu_webdav/tests/driver.py

```diff
@@ -0,0 +1,71 @@
+import os
+import hashlib
+
+from io import BytesIO
```

Comment:
I'm developing with Python 2 and it works. BytesIO is also used in the webdav library and the author says that it works with py2 and py3.